### PR TITLE
Updates & fixes

### PR DESCRIPTION
**Pull request by pjdvmalan**
* Consolidate documentation.
* Cleanup config and set sane defaults.
* Flush print to terminal.

### CODE REVIEW

File: etc/config.py [hunk 1]
```diff
@@ -109,3 +121,10 @@
         'url': 'https://yale-som-execed.onlinecampus.getsmarter.com/admin/plugins.php',
     }
 ]
+
+try:
+    # pylint: disable=unused-wildcard-import,wildcard-import
```
Comment (JJSwanevelder) [inline etc/config.py:126]: Should we leave this commented out code in? Not sure if this is intentional

File: README.md [hunk 2]
```diff
@@ -1,19 +1,46 @@
-
 # Page Checker - Basic web page checking using Selenium
 
 [![Python](https://img.shields.io/badge/Python-3-blue?logo=python&logoColor=white)](https://python.org)
 [![GitHub tag](https://img.shields.io/github/tag/pjdvmalan/page_checker?include_prereleases=&sort=semver)](https://github.com/pjdvmalan/page_checker/releases/)
 [![License - MIT](https://img.shields.io/badge/License-MIT-blue)](#license)
 
-## Installation
+## Raison d'être
```
Comment (JJSwanevelder) [inline README.md:7]: French comment added. Not sure if intentional. Do we need this here or should it be in English? "Reason for being"

**Review by JJSwanevelder — COMMENTED** [review]
Small comments added for clarification.

File: etc/config.py (hunk 1 above)
Comment (pjdvmalan) [reply etc/config.py:126]: Intentional - this is a flag to disable the warning in the linter. Wildcard required in this instance.

File: README.md (hunk 2 above)
Comment (pjdvmalan) [reply README.md:7]: Me having a moment of fun. Better translation: Reason for existence, i.e. why another Selenium test script?

**Review by JJSwanevelder — COMMENTED** [review]
👍

**Pull request merged**